### ISSUE TITLE
FindUncalledPrivateMethods: false-negative

### ISSUE THREAD

**Issue by mazze5** [spotbugs/spotbugs#2329]
Hi,

I use SpotBugs Version 4.7.3.0 with the maven-plugin.
In my case spotbugs does not recognize not used private methods in classes.

I don't follow these guildeline "https://github.com/spotbugs/spotbugs/blob/010159103cb4ad4a2fb5e0e41c8d09b31c853819/.github/CONTRIBUTING.md" adding a test, because I had problems to build the master branch locally with the gradle stuff.

Problem should be reproducable with the following minimal steps

- Create empty maven project
- add a class with unused private method for example: 
```
package org.example.test;

public class Abc {
    private void unusedMethod(){}
}
```

- run: `mvn -Dspotbugs.debug=true -Dspotbugs.trace=true com.github.spotbugs:spotbugs-maven-plugin:4.7.3.0:check`
- I used debug=true und trace=true in the command above to see that the classes was analysed and the detector "FindUncalledPrivateMethods" was called. Thats looks fine 
```
[java] Registering detector: edu.umd.cs.findbugs.detect.FindUncalledPrivateMethods
[java] Pass 1: Analyzing classes (7 / 9) - 77% complete     0 0/2  7/9 org/example/test/Abc
```

**Output:**
```
[INFO] BugInstance size is 0
[INFO] Error size is 0
[INFO] No errors/warnings found
```

**Expected Output:**
- At least a warning that method "unusedMethod" is never called

**Comment by iloveeclipse** [comment]
I see the bug reported on master (from Eclipse), just with low prio. So please consider to setup maven build to report everything, and that should do the trick.

**Comment by mazze5** [comment]
@iloveeclipse thanks. Thats it. Adding that to the maven-plugin-config solves the problem. Thanks.
<configuration>
   <threshold>Low</threshold>
</configuration>
